SpanJson/JsonWriter.Utf16.cs

```diff
@@ -378,6 +378,11 @@ public void WriteUtf16Guid(Guid value)
         }
 
         public void WriteUtf16String(string value)
```

Comment:
Hint: string has implicit conversion to ReadOnlySpan<char>, so you don't have to create overload method.
However, it's up to you how to implement this

Reply:
This is overload is for CodeGen reasons, Expression Trees don't do the implicit conversion, I noticed that for the Verbatim overloads, so I added it here too.